.github/workflows/build-container.yml

```diff
@@ -522,7 +522,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        test: ["sanity", "state_transitions", "state_transitions_both_gws", "state_transitions_loop", "state_transitions_rand_loop", "late_registration", "late_registration_loop", "4gws", "4gws_loop", "namespaces", "namespaces_loop", "mtls"]
+        test: ["sanity", "state_transitions", "state_transitions_both_gws", "state_transitions_loop", "state_transitions_rand_loop", "late_registration", "late_registration_loop", "4gws", "4gws_loop", "namespaces", "namespaces_loop", "mtls", "set_qos", "set_qos_2ms"]
     runs-on: ubuntu-latest
```

Comment:
set_qos fix was not backported to 7.1 yet, so we probably either add that fix to 7.1 or remove "set_qos", "set_qos_2ms" tests